build-deps-rhel.txt

```diff
@@ -0,0 +1,2 @@
+# Used by mantle
+golang golang-bin golang-src
```

Comment:
just add these to build-deps.txt with a comment: `golang-bin and golang-src don't get pulled in automatically on rhel`

they already get installed on fedora so it won't matter:

```
[coreos-assembler]$ rpm -qa | grep golang
golang-1.10.5-1.fc28.x86_64
golang-bin-1.10.5-1.fc28.x86_64
golang-src-1.10.5-1.fc28.noarch
```